packages/three-vrm-materials-mtoon/src/nodes/MToonLightingModel.ts

```diff
@@ -79,47 +79,52 @@ export class MToonLightingModel extends Nodes.LightingModel {
     // Unable to use `addAssign` in the current @types/three, we use `assign` and `add` instead
     // TODO: Fix the `addAssign` issue from the `@types/three` side
 
-    (reflectedLight.directDiffuse as Nodes.ShaderNodeObject<Nodes.Node>).assign(
-      (reflectedLight.directDiffuse as Nodes.ShaderNodeObject<Nodes.Node>).add(
+    (reflectedLight.directDiffuse as THREE.ShaderNodeObject<THREE.Node>).assign(
+      (reflectedLight.directDiffuse as THREE.ShaderNodeObject<THREE.Node>).add(
         getDiffuse({
           shading,
-          lightColor: lightColor as Nodes.ShaderNodeObject<Nodes.Node>,
+          lightColor: lightColor as THREE.ShaderNodeObject<THREE.Node>,
         }),
       ),
     );
 
     // rim
-    (reflectedLight.directSpecular as Nodes.ShaderNodeObject<Nodes.Node>).assign(
-      (reflectedLight.directSpecular as Nodes.ShaderNodeObject<Nodes.Node>).add(
+    (reflectedLight.directSpecular as THREE.ShaderNodeObject<THREE.Node>).assign(
+      (reflectedLight.directSpecular as THREE.ShaderNodeObject<THREE.Node>).add(
         parametricRim
           .add(matcap)
           .mul(rimMultiply)
-          .mul(Nodes.mix(Nodes.vec3(0.0), Nodes.BRDF_Lambert({ diffuseColor: lightColor }), rimLightingMix)),
+          .mul(THREE.mix(THREE.vec3(0.0), THREE.BRDF_Lambert({ diffuseColor: lightColor }), rimLightingMix)),
       ),
     );
   }
 
-  indirectDiffuse({ irradiance, reflectedLight }: Nodes.LightingModelIndirectInput) {
+  indirect(context: THREE.LightingModelIndirectInput) {
```

Comment:
not mention in https://github.com/mrdoob/three.js/wiki/Migration-Guide#166--r167 ..

Reply:
Three.js devs don't assume users to extend `NodeMaterial` 😅